fcrepo-auth-roles-common/src/test/java/org/fcrepo/auth/roles/common/AccessRolesProviderTest.java

```diff
@@ -78,14 +78,14 @@
 
     private NodeIterator rbaclIterator;
 
-    private AccessRolesProvider provider;
+    private RbAclAccessRolesProvider provider;
 
     @Before
     public void setUp() throws RepositoryException {
         initMocks(this);
         when(node.getSession()).thenReturn(session);
 
-        provider = new AccessRolesProvider();
+        provider = new RbAclAccessRolesProvider();
```

Comment:
I get the others, but this would be trying to instantiate the interface. That can't work, no?